### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by LayneXian** [community/community#102712] [Copilot Conversations]
### Select Topic Area

Question

### Body

I applied a license from my company and got feedback mail from GitHub <noreply@github.com> which told me "The owner of the organization GNCopilotAccess has granted you access to GitHub Copilot. ". But when I try to use it in VS Code, it still can't work with warnings:
[INFO] [auth] [2024-01-26T05:21:24.303Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2024-01-26T05:21:24.304Z] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as LayneXian."
What could be the problem??
![image](https://github.com/community/community/assets/8424708/1e0006fe-3de0-42e4-bf60-35dd6b92e3e9)

**Comment by LayneXian** [comment]
Should be fixed for now.

**Reply by airlanggarasyad** [reply]
how did you fix the problem?

**Reply by chenweiyi** [reply]
same ask
